documentation/docs/services/SSI-bridge/use-cases.md

```diff
@@ -0,0 +1,108 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: In the context of e-commerce and the ENSURESEC project, you can use the SSI-Bridge to verify identities for products, customer, sellers, and clients.   
+keywords:
+- ecommerce
+- scanner device
+- customer
+- credentials
+- decentralized identity
+- shopping app
+- products authenticity credentials
+---
+# Use Cases
+
+In the context of the [ENSURESEC](https://www.ensuresec.eu/) e-commerce ecosystem, the proposed use cases will use the
+Ecommerce-SSI Bridge to implement the following workflows.
+
+## Secure Goods Distribution
+
+### Delivery Company Identity and Scanners Verification
+
+**Problem**: Protection of a Delivery. Company X wants to protect its goods from being handled by unauthorized carriers
```

Comment:
"handled by unauthorized carriers, threats or frauds" as opposed to "protect from being handled, threats or frauds".

Also, Oxford comma is missing :trollface: